app/Http/Controllers/ImageController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ImageRequest;
+use App\Http\Resources\ImageResource;
+use App\Models\Page;
+use Illuminate\Http\Request;
+
+class ImageController extends Controller
+{
+    /**
+     * Get the images list filtered by the specific page
+     * 
+     * @param ImageRequest $request
+     * @param Page $pageName
```

Comment:
Doesn't looks correct. There is only one param. Probably you need to use Swagger documentation. Please check other Controllers